simple_ot2_opentrons_flow.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+"""
+Simplified Prefect + Opentrons Integration
+Focuses on getting both APIs working in same script with proper serve functionality
+"""
+
+import sys
+import os
+import subprocess
+import tempfile
+
+print("🚀 Simplified Prefect + Opentrons Integration")
+print("=" * 60)
+
+# Set up proper Prefect Cloud configuration
+print("\n🔧 Setting up Prefect Cloud configuration...")
+os.environ['PREFECT_API_URL'] = 'https://api.prefect.cloud/api/accounts/5b838504-64cf-4297-9b35-b881ac6169b3/workspaces/d2718b4c-b49a-43ce-83c2-baf6fb3b9665'
+
+# Import Prefect
+print("✅ Importing Prefect...")
+from prefect import flow, task
+import prefect
+print(f"Prefect {prefect.__version__} imported successfully")
+
+def execute_opentrons_protocol(protocol_name, protocol_code):
+    """
+    Execute Opentrons protocol in clean subprocess to avoid pydantic conflicts
+    """
+    print(f"🔬 Executing {protocol_name}...")
+    
+    # Create properly formatted Python script
+    script_content = f"""#!/usr/bin/env python3
+import sys
+sys.path.insert(0, "/usr/lib/python3.10/site-packages")
+
+try:
+{protocol_code}
+    print("SUCCESS: Protocol completed")
+except Exception as e:
+    print(f"ERROR: {{e}}")
+    sys.exit(1)
+"""
+    
+    # Write to temporary file
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.py', delete=False) as f:
+        f.write(script_content)
+        script_path = f.name
+    
+    try:
+        # Execute with system Python
+        result = subprocess.run([
+            '/usr/bin/python3', script_path
+        ], capture_output=True, text=True, timeout=30)
+        
+        if result.returncode == 0 and "SUCCESS" in result.stdout:
+            return f"✅ {protocol_name} completed successfully"
+        else:
+            error_msg = result.stderr or result.stdout
+            return f"❌ {protocol_name} failed: {error_msg[:100]}"
+            
+    except subprocess.TimeoutExpired:
+        return f"❌ {protocol_name} timed out"
+    except Exception as e:
+        return f"❌ {protocol_name} error: {e}"
+    finally:
+        # Cleanup
+        if os.path.exists(script_path):
+            os.remove(script_path)
+
+@task(name="init-robot")
+def initialize_robot():
+    """Initialize OT-2 robot using Opentrons API"""
+    
+    protocol_code = """    import opentrons.simulate
+    
+    print("🤖 Initializing OT-2 robot...")
+    
+    # Test basic simulation
+    protocol_text = '''
+metadata = {'apiLevel': '2.13'}
+
+def run(protocol):
```

Comment:
You can keep the example much simpler. I'm looking for proof of concept / minimal working example. The requirement is that a prefect task containing Opentrons code can be executed